examples/plotting/filtering.py

```diff
@@ -1,3 +1,16 @@
+''' A map representation of unemployment rate in US using the `US_Sates Dataset`_.
+This example demonstrates using IndexFilter, ColorMapper and HoverTool with 
+basic plot elements (`Area_glyphs.patches`). When hovering over the points, 
```

Comment:
```suggestion
basic plot elements such as patches. When hovering over the points, 
```
(FYI need *double* backticks for sphinx inline code formatting)

Reply:
Noted!